test/oatpp/web/FullAsyncTest.cpp

```diff
@@ -187,6 +187,11 @@ void FullAsyncTest::onRun() {
         OATPP_ASSERT(dto);
         OATPP_ASSERT(dto->testValue == "my_test_header-Async");
       }
+
+      // { // test silent connection close
+      //   auto response = client->silentClose();
```

Comment:
Client should just throw an error in this situation (I mean it should be already working:)):

```cpp
try{
      auto response = client->silentClose();
      OATPP_ASSERT(response == nullptr);
} catch (const std::runtime_erro& e) {
   ...
}
```